### PR TITLE
Updates to current live version (PR #2732 in live)

### PR DESCRIPTION
**Pull request by Georgeyoo**
Updates starter product route with latest from PR 2732 in live and adds in example .env file

### CODE REVIEW

File: Develop/.env.EXAMPLE [hunk 1]
```diff
@@ -0,0 +1,3 @@
+process.env.DB_NAME='ecommerce_db'
+process.env.DB_USER=''
+process.env.DB_PW=''
```
Comment (Xandromus) [inline Develop/.env.EXAMPLE:3]: Just a missing terminal stop 🤦

**Review by Xandromus — CHANGES_REQUESTED** [review]
Just one small thing

File: Develop/.env.EXAMPLE (hunk 1 above)
Comment (Georgeyoo) [reply Develop/.env.EXAMPLE:3]: hue hue hue

Comment (Georgeyoo) [inline Develop/.env.EXAMPLE:3]:
```suggestion
process.env.DB_PW=''

```

**Review by Xandromus — APPROVED** [review]

**Pull request merged**